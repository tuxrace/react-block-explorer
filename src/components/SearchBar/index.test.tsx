import { render } from '@testing-library/react'
import Searchbar from './index'

describe('Searchbar component', () => {
    test('renders Searchbar component', async () => {
        const {baseElement} = render(<Searchbar />);
        expect(baseElement).toMatchSnapshot()
    })
})
