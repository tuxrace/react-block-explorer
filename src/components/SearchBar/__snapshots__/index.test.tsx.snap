// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders header component 1`] = `
<body>
  <div>
    <div
      class="MuiFormControl-root MuiTextField-root makeStyles-searchStyle-1"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
        data-shrink="false"
        for="standard-start-adornment"
        id="standard-start-adornment-label"
      >
        transaction address
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
          id="standard-start-adornment"
          type="text"
          value=""
        />
        <div
          class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            ðŸ”Ž
          </p>
        </div>
        <fieldset
          aria-hidden="true"
          class="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="PrivateNotchedOutline-legendLabelled-4"
          >
            <span>
              transaction address
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
</body>
`;

exports[`Searchbar component renders Searchbar component 1`] = `
<body>
  <div>
    <div
      class="MuiFormControl-root MuiTextField-root makeStyles-searchStyle-1"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
        data-shrink="false"
        for="standard-start-adornment"
        id="standard-start-adornment-label"
      >
        transaction address
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
          id="standard-start-adornment"
          type="text"
          value=""
        />
        <div
          class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            ðŸ”Ž
          </p>
        </div>
        <fieldset
          aria-hidden="true"
          class="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="PrivateNotchedOutline-legendLabelled-4"
          >
            <span>
              transaction address
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
</body>
`;
